scout/parse/panel.py

```diff
@@ -314,11 +324,17 @@ def parse_panel_app_gene(app_gene, hgnc_map, confidence):
 
 
 def parse_panel_app_panel(
-    panel_info, hgnc_map, institute="cust000", panel_type="clinical", confidence="green"
-):
+    adapter,
```

Comment:
We have tried very hard to keep parse free from adapter/store, possibly a little overzealously so as we don't really have a huge problem with cyclic dependencies. This dose not break per se, but keeping some parts independent is part of making importing simple. Please consider if you could add the aliases in any somewhat sensible way to e.g. via the `ensembl_to_hgnc_mapping()` `hgnc_map`, or maybe via an additional `hgnc_alias_map`.